g2p/tests/test_doctor.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+from unittest import main, TestCase
+from g2p.log import LOGGER
+from g2p.mappings.langs.utils import check_ipa_known_segs
+
+class DoctorTest(TestCase):
+    def setUp(self):
+        pass
+
+    def test_ipa_known_segs_fra(self):
+        with self.assertLogs(LOGGER, level='WARNING') as cm:
+            check_ipa_known_segs(["fra-ipa"])
```

Comment:
This is awesome! I didn't know that the `self.assertLogs()` method exists!

Reply:
Yea! I didn't know about this either. I'm pretty sure all three of us have tested logs in three different ways. I think I like this the best though.

Reply:
I was going to replicate what Eddie did in his PR, but I also did a web search and found on StackOverflow another complicated method, and then a second answer lower down in the thread showed this solution, which is implemented directly for this! Way easier! I'm glad I found it. It's apparently relatively recent.